app/views/publish/courses/course_information/edit.html.erb

```diff
@@ -73,13 +78,14 @@
       </ul>
 
       <%= f.govuk_text_area(:interview_process,
+        value: @course.enrichments.last[:interview_process],
         label: { text: "Interview process (optional)", size: "s" },
         max_words: 250,
         rows: 15) %>
 
       <hr class="govuk-section-break govuk-section-break--visible govuk-section-break--l">
 
-      <h3 class="govuk-heading-m" id="how-school-placements-work"><%= @course.placements_heading %></h3>
+      <h3 class="govuk-heading-m" id="how-school-placements-work">How school placements work</h3>
```

Comment:
What's the rationale?
`@course.placements_heading` That is used a fair bit.
Do we need to remove it wholesomely?

Reply:
No, was an oversight on my part, replaced